lib/objectBuilder.js

```diff
@@ -330,6 +330,21 @@ module.exports = function () {
     return rec;
   }
 
+  function makePumpSettingsAllObjects() {
```

Comment:
Feels to me that this could just as easily be named `makeTandemPumpSettings` and it'll be accurate for quite a while if not forever. If you prefer that as well, go ahead and rename.